qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -613,6 +613,34 @@ def delete_raw_data(self, study, user, callback):
 
         callback((msg, msg_level, tab, tab_id, None))
 
+    def delete_prep_template(self, study, user, callback):
+        """Delete the selected prep template
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done
+        """
+        prep_template_id = int(self.get_argument('prep_template_id'))
+        tab_id = PrepTemplate(prep_template_id).raw_data
```

Comment:
Oh, ok. In any case using a single variable should work (I think).

On (Feb-13-15|14:17), josenavas wrote:
> > @@ -613,6 +613,34 @@ def delete_raw_data(self, study, user, callback):
> >  
> >          callback((msg, msg_level, tab, tab_id, None))
> >  
> > +    def delete_prep_template(self, study, user, callback):
> > +        """Delete the selected prep template
> > +
> > +        Parameters
> > +        ----------
> > +        study : Study
> > +            The current study object
> > +        user : User
> > +            The current user object
> > +        callback : function
> > +            The callback function to call with the results once the processing
> > +            is done
> > +        """
> > +        prep_template_id = int(self.get_argument('prep_template_id'))
> > +        tab_id = PrepTemplate(prep_template_id).raw_data
> 
> is not the `prep_id`. Is the `raw_data_id`. Also the name is tab because the callback function expect a message, the message level, the name of tab group to show, the id in that group (tab_id) and the id of the sub_tab in case that the previous ID points to another tab group.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/850/files#r24701698